### PR TITLE
  fix(FEC-8738): mobile - when there are 2 players in a page, the settings from the 2nd are presented on the first 

### PR DESCRIPTION
**Pull request by odedhutzler**
### Description of the Changes

changing portal element from id to class
changing the css accordingly

this is done so if there are 2 players, the portal will be created under the player specific player id.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/styles/_shell.scss [hunk 1]
```diff
@@ -42,14 +42,14 @@
       font-family: $font-family;
     }
   }
-  :global(#overlay-portal) {
+  :global(.overlay-portal) {
```
Comment (OrenMe) [inline src/styles/_shell.scss]: check if we can remove the global - as now we have the portal only under a player

File: src/components/smart-container/smart-container.js [hunk 2]
```diff
@@ -36,7 +37,14 @@ const mapStateToProps = state => ({
  * </SmartContainer>
  * @extends {Component}
  */
-class SmartContainer extends Component {
+class SmartContainer extends BaseComponent {
+  /**
+   * @param {Object} obj an object
+   */
+  constructor(obj: Object) {
```
Comment (OrenMe) [inline src/components/smart-container/smart-container.js]: I think it is better to keep this a regular component and pass in the player id

File: src/styles/_shell.scss (hunk 1 above)
Comment (odedhutzler) [reply src/styles/_shell.scss]: cannot remove it

File: src/components/smart-container/smart-container.js (hunk 2 above)
Comment (odedhutzler) [reply src/components/smart-container/smart-container.js]: changed.

**Pull request merged**